sql/internal/specutil/convert.go

```diff
@@ -378,6 +418,14 @@ func FromForeignKey(s *schema.ForeignKey) (*sqlspec.ForeignKey, error) {
 	}, nil
 }
 
+func schemaName(ref *schemaspec.Ref) (string, error) {
+	parts := strings.Split(ref.V, ".")
+	if len(parts) < 2 || parts[0] != "$schema" {
+		return "", fmt.Errorf("expected ref format of $schema.name")
```

Comment:
is the error clear enough? maybe add some prefix like specutil:

Reply:
good point. 
this error is always wrapped by callers that already give the package prefix. to avoid stuttering I'm keeping as is